doc/FAQ.md

```diff
@@ -164,3 +164,9 @@ Here is an example in Python checking if [one of the `option_static_remotekey` b
 >>> bool(0x02aaa2 & ((1 << 12) | (1 << 13)))
 True
 ```
+
+## Technical Questions
```

Comment:
Ops! This change make angry the GitHub action https://github.com/ElementsProject/lightning/pull/4523/checks?check_run_id=2548263926

Maybe a good place where put the FAQ it is the following one https://github.com/ElementsProject/lightning/blob/master/doc/FAQ.md ?

This make consistent all the documentation pages

Reply:
Yes, that just means that `make doc-all` needs to be called and the generated files need to be checked in. We used to not require the generated files to also be checked in, but it would result in stale docs being published.